### PR TITLE
Add synapse commons throttle utils test cases

### PR DESCRIPTION
**Pull request by manoramahp**

### CODE REVIEW

File: modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ * Test class for org.apache.synapse.commons.throttle.module.utils.Utils
+ */
+public class UtilsTest extends TestCase {
+
+    /**
+     * Test IP in range method
+     * @throws UnknownHostException
+     */
+    public void testIpInRange() throws UnknownHostException {
+        assertTrue("IP is in range", Utils.isIpInRange("192.168.1.1","192.168.1.1-192.168.1.10"));
```
Comment (chanikag) [inline modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java]: Formatting issue : Add space after ,

File: modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.commons.throttle.module.utils;
+
+import junit.framework.TestCase;
+
```
Comment (nuwanw) [inline modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java]: remove new line

File: modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java (hunk 1 above)
Comment (manoramahp) [reply modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java]: Done

File: modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java (hunk 2 above)
Comment (manoramahp) [reply modules/commons/src/test/java/org/apache/synapse/commons/throttle/module/utils/UtilsTest.java]: Done

**Pull request merged**